src/main/resources/view/PersonListPanel.fxml

```diff
@@ -3,6 +3,6 @@
 <?import javafx.scene.control.ListView?>
 <?import javafx.scene.layout.VBox?>
 
-<VBox xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
-  <ListView fx:id="personListView" VBox.vgrow="ALWAYS" />
+<VBox style="-fx-background-color: #f8f8f8ff;" xmlns="http://javafx.com/javafx/20.0.1" xmlns:fx="http://javafx.com/fxml/1">
+  <ListView fx:id="personListView" style="-fx-background-color: #f8f8f8ff;" VBox.vgrow="ALWAYS" />
 </VBox>
```

Comment:
Looking good! Might want to change the help icon in the help window but not really important tbh :D good jobb

Reply:
:D